### DISCUSSION TITLE
Not synced - Peak height lags little behind others

### DISCUSSION THREAD

**Discussion by Unwrt** [Chia-Network/chia-blockchain#3291] [Support]
Hi,

My full node does not stay synced. It used to be fully synced all the time, but now it's synced only occasionally.
I see that my peak height is lagging little bit behind others: If I'm at 211000, others might be already at 211057
Sometimes it says I'm synced even tho I see my height is ~20 behind others.. but it's starting to be "Not synced" most of the time now.

I'm running version 1.1.2 GUI on Windows 10 with 4 core Intel celeron J1900, 4gb ram and plenty of HDD space.
8444 port is open, and I seem to have ~50+ connections to other nodes, and network speed does not seem to be the problem.

Since there's pretty many connections to my node, I've been wondering if the connection count is too big - is there a way to limit it? Or if the ratio of fully synced and not synced nodes is not good enough? There's plenty of fully synced nodes connected tho. 
Also my wallet is "not synced" with 0 connections now, there used to be 1 connection before, but this might be different matter.

So can anyone tell me where the problem is? Or what should I do?

**Comment by Unwrt** [comment]
1.1.3 was released yesterday, and it seems to have solved my sync problems

**Reply by ghost** [reply]
And I have problems with sync since 1.1.3 :-/

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
